releng/org.eclipse.emfcloud.validation.codestyle/checkstyle/model-validation-checkstyle.xml

```diff
@@ -0,0 +1,276 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
can't we reuse the checkstyle from Tobias: https://github.com/eclipse-emfcloud/emfcloud/tree/master/codestyle